.gitignore

```
@@ -7,3 +7,4 @@
 __pycache__
 build/
 dist/
+.vscode/launch.json
```

We could probably just change this to ignore all files under `.vscode`.

```suggestion
.vscode/
```